test/service/test_request_validation.py

```diff
@@ -259,11 +259,15 @@ def test(url, message, params):
                  message='Missing required query parameters `entity_type`, `integration_type`')
 
     def test_bad_catalog_param(self):
-        for catalog, error in [
-            ('foo', "Catalog name 'foo' is invalid."),
-            ('foo ', "('Catalog name is invalid', 'foo ')")
-        ]:
-            url = self.base_url.set(path='/index/files', args=dict(catalog=catalog))
-            response = requests.get(str(url))
-            self.assertEqual(400, response.status_code, response.json())
-            self.assertIn(error, response.json()['Message'])
+        for path in ('/index/files',
+                     '/index/summary',
+                     '/manifest/files',
+                     '/repository/files/f1l31d'):
```

Comment:
Not a valid UUID. Since we're testing validation it might affect the accuracy of this test if there were two violations in a single request.